frontend/src/e2e-test/specs/0_citizen/citizen-daycare-application.spec.ts

```diff
@@ -236,8 +236,12 @@ describe('Citizen daycare applications', () => {
       otherGuardianPage,
       fixtures.enduserChildJariOtherGuardianFixture.ssn
     )
-    await new CitizenApplicationsPage(
-      otherGuardianPage
-    ).assertApplicationExists(applicationId)
+
+    const applications = new CitizenApplicationsPage(otherGuardianPage)
+    await applications.assertApplicationExists(applicationId)
+    const applicationReadView = await applications.viewApplication(
+      applicationId
+    )
+    await applicationReadView.contactInfoSection.waitUntilHidden()
```

Comment:
Should there first be a check that "something" is visible, and only after that a check that the contact info section is hidden?